android/src/main/java/io/sentry/react/RNSentryBreadcrumb.java

```diff
@@ -0,0 +1,87 @@
+package io.sentry.react;
+
+import com.facebook.react.bridge.ReadableMap;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Map;
+
+import io.sentry.Breadcrumb;
+import io.sentry.SentryLevel;
+
+public class RNSentryBreadcrumb {
+
+    @Nullable
+    public static String getCurrentScreenFrom(ReadableMap from) {
+        final @Nullable String maybeCategory = from.hasKey("category") ? from.getString("category") : null;
+        if (maybeCategory == null || !maybeCategory.equals("navigation")) {
+            return null;
+        }
```

Comment:
does  getString throws if not found? Otherwise we could do
```suggestion
        final @Nullable String maybeCategory = from.getString("category");
        if (!"navigation".equals(maybeCategory)) {
            return null;
        }
```